test/test.js

```diff
@@ -1,8 +1,41 @@
 const request = require('supertest');
 const app = require('../server');
+const Student = require('../src/models').Student;
 
-describe('GET /', () => {
-  it(`responds with Hello World!`, (done) => {
-    request(app).get('/').expect(`Hello World!`, done);
+describe('Sample Test', () => {
+  it('should test that true === true', () => {
+    expect(true).toBe(true)
```

Comment:
Это тоже надо наверно удалить
```suggestion
describe('Sample Test', () => {
  it('should test that true === true', () => {
    expect(true).toBe(true)
```